packages/popcorn/docs/api/utilities/interpolate.md

```diff
@@ -77,8 +77,50 @@ const mapper = interpolate(
 mapper(0.5); // '15px, rgba(128, 128, 128, 0.5)'
 ```
 
+## Options
+
+`interpolate` takes an optional third argument, options. This is an object that may contain the following configurations:
+
+### Clamp
+
+Clamp values to within the provided ranges.
```

Comment:
does this (clamping) have to be configurable? what is the use case for turning it off? Cant turning it off just be implemented in userland with `Math.max(nonClampedMax, interpolate(/**/)(/**/))`?

Reply:
The use case for turning it off is mapping between two unbounded ranges.

It could probably be achieved in userland but I think for usability most people would prefer a simple config. IMO clamping an unclamped `interpolate` is simpler and more intuitive but I also think `interpolate` will probably be used *mostly* unclamped

Reply:
> MO clamping an unclamped interpolate is simpler and more intuitive but I also think interpolate will probably be used mostly unclamped

If you think it will be used mostly unclamped then making the default of clamp true seems like a worse choice 🤔 

Im worried how this interacts with `ease` here, is the last (or first) easing used for the out of range values then? It makes some sense but according to the docs the easing function describes the easing function to be used BETWEEN sinbling input values.

Reply:
not sure if you have seen this ^, so friendly 🏓 cc @InventingWithMonster